frontend/package.json

```diff
@@ -44,10 +45,5 @@
   "engines": {
     "node": "10.x.x"
   },
-  "devDependencies": {
```

Comment:
I only needed to take out eslint to fix the problem. I think it's still a good idea to have to have the others.